priv/migrations/20221104120000_index_oracle_query_and_refunds.ex

```diff
@@ -0,0 +1,155 @@
+defmodule AeMdw.Migrations.IndexOracleQueryAndRefunds do
```

Comment:
Got this error with `testnet-1.25.1.tar.gz`:
```
warning: redefining module AeMdw.Migrations.IndexOracleQueryAndRefunds (current version loaded from _build/dev/lib/ae_mdw/ebin/Elixir.AeMdw.Migrations.IndexOracleQueryAndRefunds.beam)
  priv/migrations/20221104120000_index_oracle_query_and_refunds.ex:1

09:01:33.023 [error] CRASH REPORT Process <0.745.0> with 0 neighbours exited with reason: bad return value: {{'Elixir.AeMdw.Application',start_phase,[migrate_db,normal,[]]},{'EXIT',{function_clause,[{aeo_query_tx,oracle_id,[{ga_meta_tx,{id,account,<<96,70,239,88,30,239,116,157,73,35,96,177,84,44,123,233,151,181,221,202,13,46,81,10,67,18,178,23,153,139,252,116>>},<<43,17,244,119,202,45,27,255>>,3,50000,1000000000,80240000000000,0,{signed_tx,{aetx,oracle_query_tx,aeo_query_tx,188,{oracle_query_tx,{id,account,<<96,70,239,88,30,239,116,157,73,35,96,177,84,44,123,233,151,181,221,202,13,46,81,10,67,18,...>>},...}},...}}],...},...]}}} in application_master:init/4 line 138
```

Reply:
Done, handling for ga_meta added. Thanks!